### ISSUE TITLE
installation problem

### ISSUE THREAD

**Issue by helgasoft** [mtennekes/cols4all#15]
**remotes::install_github('mtennekes/cols4all')**
Downloading GitHub repo mtennekes/cols4all@HEAD
✔  checking for file 'C:\Users\...\AppData\Local\Temp\Rtmpyk75uW\remotes47e8476e60ad\mtennekes-cols4all-a0d3023/DESCRIPTION' (961ms)
─  preparing 'cols4all': (765ms)
✔  checking DESCRIPTION meta-information ... 
─  checking for LF line-endings in source and make files and shell scripts (451ms)
─  checking for empty or unneeded directories
─  looking to see if a 'data/datalist' file should be added
─  building 'cols4all_0.3.tar.gz'
   
Installing package into ‘C:/Users/.../Documents/R/win-library/4.2’
(as ‘lib’ is unspecified)
* installing *source* package 'cols4all' ...
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (cols4all)

**library(cols4all)**
Error: package or namespace load failed for ‘cols4all’:
 .onLoad failed in loadNamespace() for 'cols4all', details:
  call: assign("z", .z, envir = .C4A)
  error: lazy-load database 'C:/Users/.../Documents/R/win-library/4.2/cols4all/R/sysdata.rdb' **is corrupt**
In addition: Warning message:
In assign("z", .z, envir = .C4A) : internal error -3 in R_decompress1

**sessionInfo()**
R version 4.2.0 (2022-04-22 ucrt)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19043)

Basically, latest version cannot be used right now. CRAN v.0.1 is kinda old...
